QUEST.tsv

```diff
@@ -806,14 +806,14 @@ QUEST_20150317_000805	Flower Shop Lady
 QUEST_20150317_000806	People don't buy flowers anymore. {nl}The only flowers they buy are the ones to offer for the Goddesses' return. {nl}It's really depressing in many ways.
 QUEST_20150317_000807	I hope Klaipeda becomes lively like it used to be… {nl}with the fragrance of flowers in the air and people dancing in spring to worship Goddess Zemyna, {nl}the Goddess of the Earth…
 QUEST_20150317_000808	Warning
-QUEST_20150317_000809	Be careful of the Leaf Bug!{nl}They are different from Beetles. {nl}It is hard to fight them when they come in flocks so be careful. 
-QUEST_20150317_000810	Large Kepa Notice : It is not an onion!{nl}Wearing goggles is useless against Large Kepa. {nl}Watch out for powerful attacks that shake the ground.
-QUEST_20150317_000811	Please worship the image with a pious mind. You will grow stronger with {nl}the Goddess Zemyna's power, blessed from the overseer of the earth herself.
-QUEST_20150317_000812	Goddess Zemyna's Statue lies at the end of this road.
-QUEST_20150317_000813	Warning! Hanaming-infested area. {nl}Do not be fooled by their cute appearance. {nl}They attack in groups and…{nl}(Red marks are covering the letters)
-QUEST_20150317_000814	Caution! What happens to you beyond this point is your responsibility! {nl}Golems are rumored to appear in this area. Do not go any further.
-QUEST_20150317_000815	P.S. It is better to run away than to pray to the Goddess in case you encounter Golems.
-QUEST_20150317_000816	The Goddesses have disappeared. {nl}But if we fight bravely against the devils, the Goddesses will return someday. {nl} - Laimonas
+QUEST_20150317_000809	$Beware of Leaf Bugs! They are different from Beetles. {nl}It is hard to fight them when they come in flocks so be careful. 
+QUEST_20150317_000810	$Notice: Large Kepa is not an onion!{nl}Wearing goggles is useless against Large Kepa. {nl}Watch out for powerful attacks that shake the ground.
+QUEST_20150317_000811	$Please worship the Statue of the Goddess with a pious mind. You will grow stronger with {nl}the Goddess Zemyna's power, blessed by the Overseer of the Earth herself.
+QUEST_20150317_000812	Statue of Goddess Zemyna lies at the end of this road.
+QUEST_20150317_000813	$Warning! Hanaming-infested area. {nl}Do not be fooled by their cute appearance. {nl}They attack in groups and…{nl}(Red marks are covering the rest.)
+QUEST_20150317_000814	$Caution! What happens to you beyond this point is your responsibility! {nl}Golems are rumored to appear in this area. Do not go any further.
+QUEST_20150317_000815	P.S.: It's better to run away, than to pray to the Goddess in case you encounter Golems.
+QUEST_20150317_000816	The Goddesses have disappeared. {nl}But if we fight bravely against the demons, the Goddesses will return someday. {nl} - Laimonas
```

Comment:
The Goddesses have disappeared. {nl}But if we fight bravely against the demons, perhaps the Goddesses will return someday.